src/base/preferences.cpp

```diff
@@ -449,6 +451,36 @@ void Preferences::setWebUiLocalAuthEnabled(bool enabled)
     setValue("Preferences/WebUI/LocalHostAuth", enabled);
 }
 
+bool Preferences::isWebUiAuthIPBypassEnabled() const
+{
+    return value("Preferences/WebUI/EnableAuthIPBypass", false).toBool();
+}
+
+void Preferences::setWebUiAuthIPBypassEnabled(bool enabled)
+{
+    setValue("Preferences/WebUI/EnableAuthIPBypass", enabled);
+}
+
+QList<QPair<QHostAddress, int> > Preferences::getWebUiAuthBypassSubnets() const
```

Comment:
Remove unneeded space between `>`: `QList<QPair<QHostAddress, int>>` in all your code.